documentation/src/main/asciidoc/topics/proc_configuring_alpn.adoc

```diff
@@ -0,0 +1,34 @@
+[id='configure_alpn']
+= Configuring Clients for ALPN
+Configure clients to use Application-Layer Protocol Negotiation (ALPN) and
+Transport Layer Security (TLS) for {brandname} server connections.
+
+.Procedure
+
+. Configure {brandname} server endpoint connectors with encryption.
+. Enable ALPN support for Hot Rod clients that use JDK 8.
++
+JDK 9 and later supports ALPN by default. For JDK 8, you should use the
+`boringSSL` libraries that Netty provide to support ALPN.
```

Comment:
No, we now use WildFly OpenSSL for the same purpose. See https://issues.jboss.org/browse/ISPN-10803

Reply:
thanks, updated